tests/execution_spec.rs

```diff
@@ -18,7 +18,7 @@ use serde::{Deserialize, Serialize};
 use serde_json::Value;
 use tailcall::async_graphql_hyper::{GraphQLBatchRequest, GraphQLRequest};
 use tailcall::blueprint::{self, Blueprint, Upstream};
-use tailcall::cli::{init_hook_http, init_http, init_in_memory_cache, init_runtime};
+use tailcall::cli::{init_runtime, javascript};
```

Comment:
runtime api should remain consistent across the codebase. 

```
// for cli
tailcall::cli::runtime::init();

// for test
tailcall::runtime::test::init();

// for cloudflare
cloudflare::runtime::init();
```

Currently it's `init_runtime` at some places and `init` at some places.